src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -8070,6 +8070,23 @@ public void goo()
             await VerifyItemIsAbsentAsync(markup, "value");
         }
 
+        [WorkItem(54361, "https://github.com/dotnet/roslyn/issues/54361")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ConditionalAccessNullableIsUnwrappedOnParameter()
+        {
+            var markup = @"
+class A
+{
+    void M(System.DateTime? dt)
+    {
+        dt?.$$
```

Comment:
Consider adding another test without the normal member access (`dt.$$`)

Reply:
You are right. Due to the unconditionally `RemoveNullableIfPresent  `dt.$$` suggest `DateTime` members, which is wrong. Good catch.

Reply:
Test added in c1bb0ca and fix was done in f0bab57